### ISSUE TITLE
Reps Council: List all "Feedback needed" task in Panel

### ISSUE THREAD

**Issue by MichaelKohler** [firefox-addons/ideas#29]
For the Reps Council to be more efficient, it would be great to have a WebExt with a panel to view all Reps issues that currently need Council input.

This is the following GitHub query: https://github.com/mozilla/reps/issues?q=is%3Aissue+is%3Aopen+label%3A%22Council+Feedback+Needed%22 (mozilla/Reps repo, all open issued tagged with the "Council Feedback Needed" label).

I see two possible ways to solve this. Both case are a panel in the toolbar which opens a visually pleasing overview. Clicking on the title (and/or) number of the issue would open the specific GitHub issue in a new tab.

1) Write a content script for https://waffle.io/mozilla/Reps to only display the "FEEDBACK NEEDED" column in a panel
2) Write a custom HTML/JS implementation with a GitHub API query listing the issues in the panel

In both cases, it should be displayed similar to this in terms of information provided:

![screen shot 2017-07-18 at 20 04 52](https://user-images.githubusercontent.com/330324/28334035-28aed4e6-6bfa-11e7-9f59-22624b830dcb.png)

The important parts here are:

* Issue number
* Issue title
* Either of those mentioned above link to the github issue to be opened in a new tab
* Labels (doesn't need to be fancy for now, bonus if you can make it fancy in variant 2)

Bonus:
* Picture of the assignees, but as this is not important for the "Council Feedback Needed" status, this is totally bonus.

**Comment by soapdog** [comment]
working on it.

**Comment by soapdog** [comment]
Initial version available at: https://github.com/soapdog/webextension-reps-feedback-needed, **be aware that there is a bug in Firefox Nightly 56.0a1 (2017-07-18) (64-bit) on Windows 10 that prevents the add-on CSS from loading**, it works as expected on Dev Edition.

Screen capture from sample session below:
![reps-feedback](https://user-images.githubusercontent.com/23247/28341345-aca07e64-6be9-11e7-994a-4a5b4062108e.gif)

**Comment by MichaelKohler** [comment]
Great! Will test tomorrow and maybe you'll get a PR :) Definitely a nice PoC version anyway. Thank you!

**Comment by Mte90** [comment]
The extension is on AMO?

**Comment by devcer** [comment]
I don't think so. since it is made for Reps council and not end user specific, there was a discussion on telegram and finally decided not to publish it. Do you think andre should publish it?

**Comment by Mte90** [comment]
Maybe it is possible to do for all the waffle users in any case I think that issue can be closed at this point because the extension exist

**Comment by caitmuenster** [comment]
The user base for this is probably too specific for listing on AMO, but you should definitely encourage members of the Reps Council to sideload it. 

Also, nicely done @soapdog!

**Comment by soapdog** [comment]
@Mte90 AMO reviewer rejected the extension. We can always sign it and release it here on Github if people think this is a good idea.

**Comment by soapdog** [comment]
@caitmuenster thanks a lot for the kind words 😻

**Comment by MichaelKohler** [comment]
@soapdog please do, as long as we don't need to install it every time we restart the browser, I'm happy

**Comment by soapdog** [comment]
@MichaelKohler I will, just give me some days to sort it out. I will add the signed version to the releases tab on github.
